docs/xlog.md

```diff
@@ -22,21 +22,35 @@
 * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# -Xlog 
+# -Xlog
 
-To control logging to the operating system message log, use [`-Xsyslog`](xsyslog.md).
-- When [`-XX:-LegacyXlogOption`](xxlegacyxlogoption.md) is set, which is the default, the `-Xlog` option is processed for compatibility as described in the explanation.
-- When `-XX:+LegacyXlogOption` is set, `-Xlog` is equivalent to `-Xsyslog`.
+To control logging to the operating system message log, use [`-Xsyslog`](xsyslog.md). This option supersedes the legacy OpenJ9 `-Xlog` option to avoid conflicts with the new reference implementation `-Xlog` option.
 
-## Syntax
+To control the behavior of an `-Xlog` option that is specified on the command line, use [`-XX:[+|-]LegacyXlogOption`](xxlegacyxlogoption.md).
 
-        -Xlog:gc[:stderr|:file=<filename>]
+- When `-XX:-LegacyXlogOption` is set, the `-Xlog` option is processed for OpenJ9 compatibility, as described in [Compatibility processing](#compatibility-processing).
+- When `-XX:+LegacyXlogOption` is set, the `-Xlog` option can be used interchangeably with the `-Xsyslog` option. See [Legacy processing](#legacy-processing).
 
-## Explanation
 
-`-Xlog:gc` or `-Xlog:gc:stderr` is equivalent to [`-Xverbosegclog`](xverbosegclog.md). `-Xlog:gc:file=<filename>` is equivalent to `-Xverbosegclog:<filename>`. The tokens `%p` and `%t` are recognized in the `<filename>` for compatibility. `%p` is replaced with the process ID and is equivalent to the [dump agent token `%pid`](xdump.md#dump-agent-tokens). `%t` is replaced with the dump agent tokens `%Y-%m-%d_%H-%M-%S`.
+## Compatibility processing
 
-Other forms of `-Xlog` which should enable GC logging are also recognized, such as `-Xlog` or `-Xlog:all`, but since these options are intended to enable other forms of logging which are not supported, a warning is output `JVMJ9VM007W Command-line option unrecognised: <option>` in addition to enabling verbose GC logging . Similarly `-Xlog:gc:stdout` enables `-Xverbosegclog` but also outputs the warning. The `-Xverbosegclog` output goes to `stderr`, logging to `stdout` is not supported.
+Any garbage collection (GC) options passed to the `-Xlog` option are mapped to the equivalent OpenJ9 verbose GC command line options as described in the following list:
 
-<!-- ==== END OF TOPIC ==== xsyslog.md ==== -->
+- `-Xlog:gc` or `-Xlog:gc:stderr` is equivalent to [`-Xverbosegclog`](xverbosegclog.md)
+- `-Xlog:gc:file=<filename>` is equivalent to `-Xverbosegclog:<filename>`, where `<filename>` can contain the following tokens:
```

Comment:
Maybe `-Xverbosegclog:<filename>` should be `-Xverbosegclog:<updated filename>`, as `%p` and `%t` are processed before passing on the filename to `-Xverbosegclog`.